packages/design-picker/src/components/generated-design-picker.tsx

```diff
@@ -25,14 +26,22 @@ const previewImageOptions: MShotsOptions = {
 interface GeneratedDesignThumbnailProps {
 	slug: string;
 	thumbnailUrl: string;
+	isSelected: boolean;
+	onPreview: () => void;
 }
 
 const GeneratedDesignThumbnail: React.FC< GeneratedDesignThumbnailProps > = ( {
 	slug,
 	thumbnailUrl,
+	isSelected,
+	onPreview,
 } ) => {
 	return (
-		<button type="button" className="generated-design-thumbnail">
+		<button
```

Comment:
Do we need to enable `scrollable` at L51?

Reply:
Can you explain a bit more about this one? Not sure if I understood 😅

Reply:
I mean could we enable `scrollable` for MShotImage and then users can see the scrollable preview when hovering on the thumbnail. I remember @saygunnyc talked about this before or we don't need this feature as they can just click to preview the generated design.

Reply:
I think the height of the thumbnail is fixed, right?

Reply:
We might want to experiment with dynamic heights (see "if we had responsive thumbnail heights" sections in 42PgVsdlQ1v2wpk6lmmrnq-fi-1697%3A62450), which is something nice to explore once we have the basic functionality implemented 🙂

Reply:
The "fixed" means that the height is irrelevant to the dimensions of the image. If I understood correctly, the responsive heights are related to the window height instead of the image dimensions, right? But maybe enabling the scrollable option is out of scope. We could do it later if we need this feature.